### PR TITLE
Added PLASMAGUARD flag to some spacesuits

### PR DESCRIPTION
**Pull request by Rei1226**
Added the PLASMAGUARD flag to ERT suits, deathsquad suits, and CE/Atmos suits as requested in 

https://github.com/d3athrow/vgstation13/issues/2369

### CODE REVIEW

File: code/modules/clothing/spacesuits/ert.dm [hunk 1]
```diff
@@ -8,6 +8,7 @@
 	max_heat_protection_temperature = FIRE_HELMET_MAX_HEAT_PROTECITON_TEMPERATURE
 	var/obj/machinery/camera/camera
 	species_restricted = list("exclude","Vox")
+	flags = PLASMAGUARD
```
Comment (d3athrow) [inline code/modules/clothing/spacesuits/ert.dm]: flages = FPRINT | TABLEPASS | HEADCOVERSEYES | BLOCKHAIR | STOPSPRESSUREDMAGE | PLASMAGUARD

File: code/modules/clothing/spacesuits/ert.dm [hunk 2]
```diff
@@ -36,6 +37,7 @@
 	/obj/item/weapon/screwdriver, /obj/item/weapon/weldingtool, /obj/item/weapon/wirecutters, /obj/item/weapon/wrench, /obj/item/device/multitool, \
 	/obj/item/device/radio, /obj/item/device/analyzer, /obj/item/weapon/gun/energy/laser, /obj/item/weapon/gun/energy/pulse_rifle, \
 	/obj/item/weapon/gun/energy/taser, /obj/item/weapon/melee/baton, /obj/item/weapon/gun/energy/gun)
+	flags = PLASMAGUARD
```
Comment (d3athrow) [inline code/modules/clothing/spacesuits/ert.dm]: flags = FPRINT | TABLEPASS | STOPSPRESSUREDMAGE | PLASMAGUARD

**Comment by Rei1226** [comment]
^- Fixed

**Comment by d3athrow** [comment]
i figured you'd get the hint on the other ones

File: code/modules/clothing/spacesuits/rig.dm [hunk 3]
```diff
@@ -58,6 +58,7 @@
 	_color = "white"
 	species_restricted = list("exclude","Vox")
 	max_heat_protection_temperature = FIRE_HELMET_MAX_HEAT_PROTECITON_TEMPERATURE
+	flags = PLASMAGUARD
```
Comment (d3athrow) [inline code/modules/clothing/spacesuits/rig.dm]: This and all the flags = plasmaguard below.

**Comment by Rei1226** [comment]
now fixed

File: code/modules/clothing/spacesuits/rig.dm [hunk 4]
```diff
@@ -66,6 +67,7 @@
 	desc = "An advanced suit that protects against hazardous, low pressure environments. Shines with a high polish."
 	item_state = "ce_hardsuit"
 	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECITON_TEMPERATURE
+	flags = FPRINT | TABLEPASS | HEADCOVERSEYES | BLOCKHAIR | STOPSPRESSUREDMAGE | PLASMAGUARD
```
Comment (ComicIronic) [inline code/modules/clothing/spacesuits/rig.dm]: These are not the flags for a spacesuit, I do not believe.

Comment (Rei1226) [inline code/modules/clothing/spacesuits/rig.dm]: Don't see why they wouldn't work just fine for a spacesuit?

Comment (ComicIronic) [inline code/modules/clothing/spacesuits/rig.dm]: You copied and pasted them from helmet. Several of these flags, like BLOCKHAIR and HEADCOVERSEYES, are only intended for hats.
The rig suit inherits its flags from /obj/item/clothing/suit/space, which has 
`    flags = FPRINT  | STOPSPRESSUREDMAGE`
This is what should be before plasmaguard.

**Pull request closed without merging**